src/components/chrome/ChromeFields.jsx

```diff
@@ -87,10 +86,6 @@ class ChromeFields extends ReactCSS.Component {
     };
   }
 
-  handleChange(data) {
-    this.props.onChange(data);
-  }
-
```

Comment:
@rickyvetter yeah when you define the same thing twice, it fires the later one, so the one left is definitely the one that has been running.